meta_modules/setup_meta_modules.py

```diff
@@ -650,3 +650,5 @@ def substitute_config_vars(config_str):
                 with open(python_module_file, 'w') as f:
                     f.write(module_content)
                 logging.info("  ... writing {}".format(python_module_file))
+
+logging.info("Metamodule generation completed successfully in {}".format(meta_module_dir))
```

Comment:
This is nice to have, because one can copy & paste the `meta_module_dir` for the `module use` statement.